RedBugs_webapp/src/main/java/com/isa/pl/redbugs/Application.java

```diff
@@ -11,20 +12,22 @@
 @SpringBootApplication
 public class Application {
 //a
-	public static void main(String[] args) {
-		SpringApplication.run(Application.class, args);
-	}
+public static void main(String[] args) {
+	SpringApplication.run(Application.class, args);
+}
 
 	@Bean
 	public CommandLineRunner commandLineRunner(ApplicationContext ctx) {
```

Comment:
to jest generalnie zbędne

Reply:
Usunięte

Reply:
Szukałem w necie i żeby mieć stronę statyczną, oraz zarządzaną poprzez bootstrap to chyba musi być w osobnych folderach, stąd mam to w folderze templates w pod folderze static. Jak dojdziemy do tego, że każda strona będzie zarządzana poprzez boostrap to trzeba to przenieść o poziom wyżej z static do tempaltes